### PR TITLE
enh(release): add Centreon Web 20.10.14 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,19 @@ If you have feature requests or want to report a bug, please go to our
...
 ## Centreon Web
 
+### 20.10.14
+
+#### Bug fixes
+
+- [Authentication] Fixed PHP error when debug is enabled with OIDC authentication
+- [CLAPI] Fixed LDAP configuration ID for export and import
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
+- [Configuration] Fixed Centreon Broker configuration that could be empty or deleted
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:29]: This line is duplicated

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,19 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Core] REmove PHP warning in logs
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:30]: Remove*

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,19 @@ If you have feature requests or want to report a bug, please go to our
...
+- [Resources Status] Fixed access to action URLs
+- [Resources Status] Fixed access to actions buttons
+- [UX] Non admin user do not have the same submenu subsections
+
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:34]: Maybe MON-11139 : [Dependency to upgrade on : centreon/centreon] - composer prod. bump nelmio/cors-bundle from 1.5.6 to 2.1.1 should appear

**Review by mdebauchez — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md (hunk 3 above)
Comment (lpinsivy) [reply en/releases/centreon-core.md:33]: no this is technical

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,18 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.14
+
```
Comment (chgautier) [inline en/releases/centreon-core.md:23]:
```suggestion

Release date: `November 29, 2021`

```

File: fr/releases/centreon-core.md [hunk 5]
```diff
@@ -19,6 +19,18 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.14
+
```
Comment (chgautier) [inline fr/releases/centreon-core.md:23]:
```suggestion

Release date: `November 29, 2021`

```

**Review by chgautier — CHANGES_REQUESTED** [review]

**Review by chgautier — APPROVED** [review]

**Pull request merged**